pkg/kubelet/kubelet_network_linux.go

```diff
@@ -76,22 +90,22 @@ func (kl *Kubelet) syncNetworkUtil(iptClient utiliptables.Interface) bool {
 	// Setup KUBE-MARK-DROP rules
 	dropMark := getIPTablesMark(kl.iptablesDropBit)
 	if _, err := iptClient.EnsureChain(utiliptables.TableNAT, KubeMarkDropChain); err != nil {
-		klog.Errorf("Failed to ensure that %s chain %s exists: %v", utiliptables.TableNAT, KubeMarkDropChain, err)
+		klog.ErrorS(err, "Failed to ensure that nat table exists KUBE-MARK-DROP chain", "natTable", utiliptables.TableNAT)
```

Comment:
Ok, I think that nat table is also const https://github.com/kubernetes/kubernetes/blob/63f89493ec3c03c3cc9681e86d13da31cb25b2dd/pkg/util/iptables/iptables.go#L105

Can someone who know this code help us decide which version is better:

```suggestion
		klog.ErrorS(err, "Failed to ensure that nat chain exists KUBE-MARK-DROP chain")
```
or something like
```suggestion
		klog.ErrorS(err, "Failed to ensure that chain exists in table", "table", utiliptables.TableNAT, "chain", KubeMarkDropChain)
```

/cc @ehashman 
Elena, do you know someone who can help here? I looked through git history and those logs are very old.